cli/cli_test.go

```diff
@@ -449,8 +515,6 @@ range_max_bytes: 67108864
 	// - attrs: [us-west-1b, ssd]
 	// range_min_bytes: 8388608
 	// range_max_bytes: 67108864
-	// gc:
```

Comment:
I'm confused. Looks like you commented out the whole test (which is fine) but this change to the test body looks like a rebase error.